emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml

```diff
@@ -0,0 +1,17 @@
+old_sound_rabbit_mq:
+    connections:
+        default:
+            url: '%env(RABBITMQ_URL)%'
+            vhost: '%env(RABBITMQ_VHOST)%'
+            lazy:     false
```

Comment:
Could we set this to true? 

That is a great performance boost =)

Reply:
Fixed.